### ISSUE TITLE
How to install the program?

### ISSUE THREAD

**Issue by manussn** [melexis/mlx90640-library#96]
Hi, 

I don't understanding with work it's program.
I use arduino. So I need copy the code in the program?
I couldn't save the library in .zip.

My project is build a infrared Thermography camera with arduino and two lamps.

Please help me!

**Comment by LBuydens** [comment]
Yes, you need to copy this sample code in your own program, add there all necessary interfaces for the arduino and run it from there.
